api/functions/process_dataset.py

```diff
@@ -62,7 +71,40 @@ def callback(publish_future: pubsub_v1.publisher.futures.Future) -> None:
     print(f"Published messages with error handler to {topic_path}.")
 
 
-def process_dataset_file(event, context):
+def clean_up_options(dataset_options: Dict) -> Dict:
```

Comment:
Nit: Dict is equivalent to Dict[Any, Any], just checking we can't be any more specific here.

Reply:
I think for the input dictionary, if I tried to precisely map out the type, I would have something which is like 10+ tiers deep because of the value padding that firestore events give. I think for the output dictionary, the cleanest thing to do is to make it Dict[str, Any], which is what I've changed it to in the next revision.